com.trollworks.gcs/src/com/trollworks/gcs/preferences/ReferenceLookupPreferences.java

```diff
@@ -73,7 +96,12 @@ public ReferenceLookupPreferences(PreferencesWindow owner) {
             idLabel.setOpaque(true);
             idLabel.setBackground(background);
             mPanel.add(idLabel, new PrecisionLayoutData().setFillHorizontalAlignment());
-            EditorField field = new EditorField(new DefaultFormatterFactory(new IntegerFormatter(-9999, 9999, true)), event -> ref.setPageToIndexOffset(((Integer) event.getNewValue()).intValue()), SwingConstants.RIGHT, Integer.valueOf(ref.getPageToIndexOffset()), Integer.valueOf(-9999), I18n.Text("If your PDF is opening up to the wrong page when opening page references, enter an offset here to compensate."));
+            EditorField field = new EditorField(new DefaultFormatterFactory(new IntegerFormatter(-9999, 9999, true)),
```

Comment:
I don't believe any changes were needed to this line. Assuming not, please revert.

Reply:
This is a strange diff, but I broke that line into five lines because it was extraordinarily long. I don't toe the 80-characters-or-bust line very strictly, but I usually will add line breaks if I have to scroll to read a line.